service/history/timerQueueProcessor.go

```diff
@@ -164,10 +164,7 @@ func (t *timerQueueProcessorImpl) NotifyNewTimers(
 	timerTasks []tasks.Task,
 ) {
 
-	if clusterName == cluster.FakeClusterForEmptyVersion {
-		return
-	}
-	if clusterName == t.currentClusterName {
+	if clusterName == t.currentClusterName || clusterName == cluster.FakeClusterForEmptyVersion {
```

Comment:
`clusterName == cluster.FakeClusterForEmptyVersion`

there should be a dedicated passive task processing logic for this right?
so this is not for active